src/pages/PingDetail.svelte

```diff
@@ -3,10 +3,73 @@
 
   export let params;
   const pingDataPromise = getPingData(params.app, params.ping);
+  const URL = `data/${params.app}/index.json`;
+  let app;
+
+  fetch(URL)
+    .then((r) => r.json())
+    .then((ret) => {
+      app = ret;
+    });
```

Comment:
This should not be necessary-- a short metric summary should also be sent as part of the ping.

I just realized that this is not currently the case though, which is a good opportunity to get some experience with the python parts of the codebase.

Reply:
Thanks for the review, As far as I understand I have to modify build-glean-metadata.py so that it writes pings data in a way that each ping should include 

- A list of metrics associated with the ping.
- A short description of each metric

This will help us avoid re-downloading the whole app data.

Reply:
Should update the build-glean-metadata.py in a separate PR?

Reply:
go ahead and do it in this one!